pkg/controllers/user/networkpolicy/netpol.go

```
@@ -132,7 +141,16 @@ func (npmgr *netpolMgr) programNetworkPolicy(projectID string) error {
 	return nil
 }
 
-func (npmgr *netpolMgr) hostPortsUpdateHandler(pod *corev1.Pod) error {
+func (npmgr *netpolMgr) hostPortsUpdateHandler(pod *corev1.Pod, clusterName string) error {
+	systemNamespaces, err := npmgr.getSystemNamespaces(clusterName)
+	if err != nil {
+		logrus.Errorf("netpolMgr: hostPortsUpdateHandler: getSystemNamespaces: err=%v", err)
+		return nil
```

why do we ignore the error? We should throw it up